javav2/example_code/dynamodb/src/main/java/com/example/enhanced/dynamodb/DynamoDBMappingGetItem.java

```diff
@@ -0,0 +1,165 @@
+import software.amazon.awssdk.services.dynamodb.model.ProvisionedThroughput;
+// snippet-end:[dynamodb.java2.enhanced.getitem.import]
+
+public class DynamoDBMappingGetItem {
+
+    private static final ProvisionedThroughput DEFAULT_PROVISIONED_THROUGHPUT =
+            ProvisionedThroughput.builder()
+                    .readCapacityUnits(50L)
+                    .writeCapacityUnits(50L)
+                    .build();
+
+    private static final TableSchema<Record> TABLE_SCHEMA =
+            StaticTableSchema.builder(Record.class)
+                    .newItemSupplier(Record::new)
+                    .addAttribute(String.class, a -> a.name("id")
+                            .getter(Record::getId)
+                            .setter(Record::setId)
+                            .tags(primaryPartitionKey()))
+                    .addAttribute(String.class, a -> a.name("sort")
+                            .getter(Record::getSort)
+                            .setter(Record::setSort)
+                            .tags(primarySortKey()))
+                    .addAttribute(String.class, a -> a.name("attribute")
+                            .getter(Record::getAttribute)
+                            .setter(Record::setAttribute))
+                    .addAttribute(String.class, a -> a.name("attribute2*")
+                            .getter(Record::getAttribute2)
+                            .setter(Record::setAttribute2)
+                            .tags(secondaryPartitionKey("gsi_1")))
+                    .addAttribute(String.class, a -> a.name("attribute3")
+                            .getter(Record::getAttribute3)
+                            .setter(Record::setAttribute3)
+                            .tags(secondarySortKey("gsi_1")))
+                    .build();
+
+
+
+    public static void main(String[] args) {
+
+         //Create a DynamoDbClient object
+        Region region = Region.US_EAST_1;
+        DynamoDbClient ddb = DynamoDbClient.builder()
+                .region(region)
+                .build();
+
+       try{
+           // snippet-start:[dynamodb.java2.enhanced.getitem.main]
+           // Create a DynamoDbEnhancedClient and use the DynamoDbClient object
+           DynamoDbEnhancedClient enhancedClient = DynamoDbEnhancedClient.builder()
+                    .dynamoDbClient(ddb)
+                    .build();
+
+           //Create a DynamoDbTable object
+           DynamoDbTable<Record> mappedTable = enhancedClient.table("Record", TABLE_SCHEMA);
+
+           //Create a KEY object
+           Key key = Key.builder()
+                   .partitionValue("id-value")
+                   .sortValue("sort-value")
+                   .build();
+
+           // Create a GetItemEnhancedRequest object
+           GetItemEnhancedRequest getItemReq = GetItemEnhancedRequest.builder()
+                   .key(key)
+                   .build();
+
+           Record result = mappedTable.getItem(getItemReq);
```

Comment:
Would it be better to demonstrate the short-form: mappedTable.getItem(key) ?

Reply:
Done - i updated the code 

 //Create a KEY object
           Key key = Key.builder()
                   .partitionValue("id-value")
                   .sortValue("sort-value")
                   .build();

           // Get the item by using the key            
           Record result = mappedTable.getItem(r->r.key(key));